index.html

```diff
@@ -18,6 +18,12 @@ <h2>My Logo</h2>
             <nav class="menu">
               <nav><img src="photo/navs/Union.png" alt=""></nav>
             </nav>
+
+            <div class="desktop-nav-bars">
+              <span>Portfolio</span>
```

Comment:
- Please work with the font-size, spacing, and background image alignment as specified in the Figma design. So that your design looks similar to that of the Figma design.  

##### Your design 
![Screenshot from 2022-07-16 09-25-28](https://user-images.githubusercontent.com/58771507/179347202-00b3959e-adf3-4a13-b42f-87e0784b3f1c.png)

##### Figma design


![Screenshot from 2022-07-16 09-29-19](https://user-images.githubusercontent.com/58771507/179347223-4c248ddc-8a72-4226-ab36-6bc65d956bdb.png)